spotify.md

```diff
@@ -0,0 +1,118 @@
+# 10 tips for maintaining a long-term relationship with docs like code
+
+##### By Gary Niemen, Product Manager for TechDocs, Spotify’s docs-like-code solution
+
+I remember about two years ago sitting here (well, not precisely “here” because we sat in offices in those days) watching, wide-eyed and a bit confused, a video of [Riona MacNamara’s talk from Write the Docs 2015](https://youtu.be/EnB8GtPuauw). In the talk, Riona described how she and another technical writer used a docs-like-code approach to change Google engineering culture around technical documentation.
+
+I remember seeing this adoption curve and saying to myself: We need to do this. I want that curve.
+
+![image](./images/adoption1.png)
+
+And just a few months later, we had built and released TechDocs, Spotify’s docs-like-code solution. And a few months after that, we had the curve that I had longed for. Nice gradient, eh?
+
+![image](./images/adoption2.png)
+
+We were fortunate in that there were several synchronous moments that helped us, but overall, we moved fast because we were able to make use of preexisting infrastructure, such as [Backstage](https://backstage.io/) (our homegrown and now open sourced developer portal) and Tingle, Spotify’s CI/CD tool. We also made use of the open source static site generator, [MkDocs](https://www.mkdocs.org/).
+
```

Comment:
Just to make sure I understand, you're building your docs to https://developer.spotify.com/ correct? Let's add a link to that here also and talk about your audience(s).. are they both internal and external developers? Do you also help teams write for internal tools?

Reply:
That's a really good question and something I have forgotten to mention. Its all internal documentation - Spotify infrastructure engineering teams supporting Spotify feature engineering teams. So everything ends up in our internal developer portal known as Backstage - now open sourced. In fact the TechDocs frontend is a so-called Plugin within Backstage. So not developer.spotify.com. We should probably clarify that it is internal documentation that we are talking about. You could add "for internal technical documentation" at the end of...
"And just a few months later, we had built and released TechDocs, Spotify’s docs-like-code solution <...>."